packages/minimongo/sorter.js

```diff
@@ -280,12 +280,6 @@ export default class Sorter {
       }
     });
 
-    // This could happen if our key filter somehow filters out all the keys even
```

Comment:
Motivation: this was no longer necessary after https://github.com/meteor/meteor/pull/10214 since there's no longer a key filter.